src/MAUI/Maui.Samples/Samples/Data/GeodatabaseTransactions/GeodatabaseTransactions.xaml.cs

```diff
@@ -105,53 +109,53 @@ private async Task GetLocalGeodatabase()
                         {
                             Microsoft.Maui.ApplicationModel.MainThread.BeginInvokeOnMainThread(() =>
                             {
-                                // Hide the progress control and report the exception
+                                // Hide the progress control and report the exception.
                                 LoadingProgressBar.IsVisible = false;
                                 MessageTextBlock.Text = "Unable to create local geodatabase: " + generateGdbJob.Error.Message;
                             });
                         }
                     };
 
-                    // Start the generate geodatabase job
+                    // Start the generate geodatabase job.
                     _localGeodatabase = await generateGdbJob.GetResultAsync();
                 }
             }
             catch (Exception ex)
             {
-                // Show a message for the exception encountered
+                // Show a message for the exception encountered.
                 Microsoft.Maui.ApplicationModel.MainThread.BeginInvokeOnMainThread(() =>
                 {
                     Application.Current.MainPage.DisplayAlert("Generate Geodatabase", "Unable to create offline database: " + ex.Message, "OK");
```

Comment:
We are catching this exception on MacCatalyst, sample is unusable. Can also repro on v.next.

Reply:
![Screenshot 2023-08-02 at 1 47 15 PM](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/assets/118313312/6852ad48-add5-47f5-b5fd-42ea76d1e2b8)

Reply:
Fixed with [commit](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/pull/1274/commits/defcbce79b2a9bcf67a1dcef7a08e399ae4a66b8).